### DISCUSSION TITLE
GitHub Enterprise support?

### DISCUSSION THREAD

**Discussion by Sam13** [Cian911/gomerge#49] [Q&A] [answered]
Does _gomerge_ support GitHub enterprise installations?
I couldn't find any parameter to pass the base URL.

**Comment by Cian911** [comment]
~Hi @Sam13 no it doesn't currently, but it's certainly something I can look into adding. Can you make an issue and I'll take a look?~

@Sam13 Doubling back on this after taking a look, I don't have an enterprise installation to test, but once you have a `GITHUB_TOKEN` with access to the org, Gomerge _should_ work as normal. Can you confirm if this is the case?

**Reply by Sam13** [reply]
Hi @Cian911 
I'm talking about GitHub Enterprise installations which are hosted on a custom domain - not on github.com.
So it must be possible to pass the base URL to gomerge and then it should work as normal.

**Reply by Cian911** [reply]
Thanks @Sam13 I've created a draft PR with what I think will be the initial foundations to support this which you [can find here](https://github.com/Cian911/gomerge/pull/53). Can you checkout my branch there and build `gomerge` to test?

You can use the project Makefile to build and run the binary as needed. Thanks!

**Reply by Sam13** [reply]
@Cian911 Thanks.
Can confirm that it works with GHE.
Sucessfully listed and merged pull requests.

**Reply by Cian911** [reply]
Great to hear @Sam13 - I will tidy up that PR and will try and get a new release out this week.

Also please give us a star if you feel so inclined, it helps me out :)

**Comment by Cian911** [comment, marked as answer]
Hi @Sam13 support for this is now released as part of the `3.1.0` release. Thanks! https://github.com/Cian911/gomerge/releases/tag/3.1.0
